### ISSUE TITLE
accessible dark theme

### ISSUE THREAD

**Issue by tonyfast** [Quansight-Labs/jupyterlab-accessible-themes#55]
this package is great. thanks for making it. it is cool to see more canonical systems joining the package. 

there is an immediate need a recommended accessible dark theme for the syntax highlighting. this need is raised by a recent notebook v7 audit https://github.com/jupyter/notebook/issues/6929

is there a dark theme on roadmap that might be more appropriate as base theme? could it provide some recommendations that might immediately improve the dark them in v7?

**Comment by steff456** [comment]
Yes! Right now we have one PR for adding the Github dark theme, after this one is merged and other issues are fixed we will make another release. 

I do also want to fix the issues that are present in the audit, for that we will need to work on creating a full themes for Jupyter light/dark. Right now we have around 6 accessible themes that we want to add before working on this, so we may take a while to make this happen.
